cmd/dep/testdata/init/case2/manifest.golden.json

```diff
@@ -0,0 +1,7 @@
+{
+    "dependencies": {
+        "github.com/sdboyer/deptest": {
+            "version": ">=0.8.0, <1.0.0"
+        }
```

Comment:
Yeah, I don't know what's going on here. Looking at how this is set up, it seems like this manifest really should have an entry for both deptest and deptestdos. Let's make sure we figure out why this is happening before merging - perhaps we've found a bug.